.devcontainer/scripts/notify-dev-entrypoint.sh

```diff
@@ -8,19 +8,18 @@ set -ex
 ###################################################################
 
 # Define aliases
-echo -e "\n\n# User's Aliases" >> ~/.bashrc
-echo -e "alias fd=fdfind" >> ~/.bashrc
-echo -e "alias l='ls -al --color'" >> ~/.bashrc
-echo -e "alias ls='exa'" >> ~/.bashrc
-echo -e "alias l='exa -alh'" >> ~/.bashrc
-echo -e "alias ll='exa -alh@ --git'" >> ~/.bashrc
-echo -e "alias lt='exa -al -T -L 2'" >> ~/.bashrc
+echo -e "\n\n# User's Aliases" >> ~/.zshrc
+echo -e "alias fd=fdfind" >> ~/.zshrc
+echo -e "alias l='ls -al --color'" >> ~/.zshrc
+echo -e "alias ls='exa'" >> ~/.zshrc
+echo -e "alias l='exa -alh'" >> ~/.zshrc
```

Comment:
Did you have a chance to notice the different `exa` output? (wondering if you know the tool and how it differs from regular 'ls')